bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -78,47 +76,43 @@ public SyncThread(ServerConfiguration conf,
             .setNameFormat("SyncThread-" + conf.getBookiePort() + "-%d");
         this.executor = Executors.newSingleThreadScheduledExecutor(tfb.build());
         flushInterval = conf.getFlushInterval();
```

Comment:
this 'flushInterval' conf is not used anywhere anymore. This conf can be removed and correspondingly all its callers in testcases.

Reply:
I will mark it as deprecated and remove it the release after next one.